src/components/add-account-dialog/add-account-dialog.scss

```diff
@@ -3,6 +3,14 @@
     top: inherit !important;
     left: inherit !important;
   }
+  &__input-ripple-error,
```

Comment:
please recheck these classes -- because Kseniia and Kostya did not use additional styles for validation